src/promptflow/promptflow/_sdk/entities/_flow.py

```diff
@@ -142,37 +156,39 @@ def path(self) -> Path:
             )
         return flow_file
 
-    @property
-    def context(self) -> FlowContext:
-        return self._context
-
-    @context.setter
-    def context(self, val):
-        if not isinstance(val, FlowContext):
-            raise UserErrorException("context must be a FlowContext object, got {type(val)} instead.")
-        self._context = val
+    @classmethod
+    def _is_dag_flow(cls, data: dict):
```

Comment:
@wangchao1230  shall we add a field (e.g. type) to distinguish eager flow?

Reply:
Is the yaml schema easy to describe the mutual exclusion of path and nodes?

Reply:
Offline synced, it's not easy to decide what value should we put here. So we currently use "entry" to decide if it's a eager flow.